start.sh

```diff
@@ -242,6 +246,10 @@ while [[ $# -gt 0 ]]; do
         paymail_domain="$2"
         shift
         ;;
+        -e|--expose)
+        expose="true"
+        # no additional arguments so now `shift` command
```

Comment:
`no` instead of `now` I think. Also 3 lines below

Reply:
Changed to `-e true` instead of simple switch